widgy/models/versioning.py

```diff
@@ -193,9 +193,18 @@ def get_owner_related_names(cls):
 
     @cached_property
     def owners(self):
-        return list(owner
-                    for attr in self.get_owner_related_names()
-                    for owner in getattr(self, attr).all())
+        acc = []
+        for attr in self.get_owner_related_names():
+            manager = getattr(self, attr)
```

Comment:
It looks like you're trying to bypass CurrentSiteManager. I'd expect _base_manager to be used for that and have the filtering happen somewhere else. I don't think it's appropriate to have owners not actually return all owners.